auth-web/src/components/auth/manage-business/ExpandableHelp.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <div class="help-btn top mb-8" @click="helpToggle = !helpToggle">
+      <v-icon color="primary">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle" class="pl-2">Help with Starting a Business</span>
+      <span v-else class="pl-2">Hide Help</span>
+    </div>
+    <v-expand-transition>
+      <div v-show="helpToggle" class="help-section pa-4 my-3 mb-8">
+        <slot name="content"></slot>
+        <div class="help-btn bottom text-right d-flex" @click="helpToggle = !helpToggle">Hide Help</div>
```

Comment:
If the Vuetify classes don't work here, please remove them.

Reply:
Got it, will remove for the next commit